src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationServiceTest.kt

```diff
@@ -1041,7 +1041,7 @@ class DraftAdjudicationServiceTest {
   inner class InProgressDraftAdjudications {
     @BeforeEach
```

Comment:
We ought to have a test for the new method, but it is so simple that I can't see any point.

However, an int test might be good?!